testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergPartitionEvolution.java

```diff
@@ -43,19 +45,19 @@ public void testDroppedPartitionField(boolean dropFirst)
 
         onSpark().executeQuery("ALTER TABLE iceberg_test.default.test_dropped_partition_field DROP PARTITION FIELD " + (dropFirst ? "a" : "b"));
 
-        assertThat(onTrino().executeQuery("SHOW CREATE TABLE test_dropped_partition_field"))
-                .containsOnly(
-                        row("CREATE TABLE iceberg.default.test_dropped_partition_field (\n" +
+        Assertions.assertThat((String) getOnlyElement(getOnlyElement(onTrino().executeQuery("SHOW CREATE TABLE test_dropped_partition_field").rows())))
```

Comment:
static import? not possible for some reason?

Reply:
in PT, we use tempto's `assertThat`, so assertj's (rare) ends up without static import